### ISSUE TITLE
Profile MFE Accessibility Evaluation

### ISSUE THREAD

**Issue by sarina** [openedx/frontend-app-profile#715]
## Goal

Produce a report with an assessment of this feature or repo's compliance with WCAG 2.1 at an AA level. If possible, WCAG 2.2 should be used instead of 2.1.

## Report Format

1. Overall assessment: is this feature Green (at least ~90% compliant), Yellow (> ~70% compliant), or Red (< 70% compliant). This is a rough assessment and should be based on the overall feel of the feature.
2. Provide an enumerated list of accessibility issues with the feature. Where possible, provide recommendations as to how to address major holes or links to relevant documentation

This work will be collated with other, similar tickets that are addressed as part of [FC-0017](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3692920833/FC-0017+-+General+Accessibility+Projects).
